system/settings/settings.xml

```diff
@@ -345,10 +345,33 @@
           </constraints>
           <control type="spinner" format="string" />
         </setting>
-        <setting id="videolibrary.jumptofirstunplayeditem" type="boolean" label="21416" help="21466">
+        <setting id="videolibrary.tvshowselectfirstunwatcheditem" type="integer" label="21416" help="21466">
```

Comment:
I thought about putting it in by didn't really like the double "s" that resulted, looked like a typo ;)

Will put it back